### DISCUSSION TITLE
What event bridge event is triggered when a pipeline concatenation is complete?  I cant find any docs on it.

### DISCUSSION THREAD

**Discussion by Demi-ob** [aws/aws-sdk-js#4484] [Q&A]
I am working on implementing the AWS chime pipeline concatenation to merge videos/audio recorded from a chime session.
What Event bridge event is triggered when the concatenation is completed successfully? I don't see anything in the docs on it

https://docs.aws.amazon.com/chime-sdk/latest/dg/create-concat-pipe.html

The docs I [referenced](https://docs.aws.amazon.com/chime-sdk/latest/dg/create-concat-pipe.html) is making use of `aws-sdk` in nodejs to create a MediaConcatenatePipeline on the backend.
e.g 
```
const chimeWithMediaPipeline = new AWS.ChimeSDKMediaPipelines({
  region: "us-east-1",
});
const createdMediaPipelineConcat = await chimeWithMediaPipeline
    .createMediaConcatenationPipeline(mediaConcatenatePipelineParams)
    .promise();
```

Everything else works fine, however there is no documentation anywhere on the Event Bridge event that gets triggered when a MediaCapturePipeline is completed (or any state change). 
Therefore I have no way to know that a MediaCapturePipeline is complete so I can update the record state in my own database.

For the MediaCapturePipeline (Not to be mistaken with MediaConcatenatePipeline), 
i.e
```
const createdMediaPipeline = await chimeWithMediaPipeline
    .createMediaCapturePipeline(mediaCapturePipelineParams)
    .promise();

```
I have an EventBridge listener that that enqueues an SQS on event or state change i.e 
 ` "chime:MediaPipelineInProgress"  "chime:MediaPipelineDeleted" "chime:MediaPipelinePermanentFailure"`
 Then I have a listener in my backend application on the SQS so I can process the different state changes.
 
 I could find any resource for this on `createMediaConcatenationPipeline` in any documentation or even in the EventBridge templates

**Basically How do i know that a MediaConcatenationPipeline is complete, failed or state change?**
